### PR TITLE
docs(autoware-interfaces): fix description of base link

### PR DESCRIPTION
**Pull request by isamu-takagi**
Signed-off-by: Takagi, Isamu <isamu.takagi@tier4.jp>

## Description

Fix  description of base link.
https://autowarefoundation.github.io/autoware-documentation/main/design/autoware-architecture/localization/#tf-tree

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The Reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [x] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: docs/design/autoware-interfaces/components/vehicle-dimensions.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 ![Vehicle Axes](images/vehicle_axes.svg){: style="width:500px"}
 
-The `base_link` frame is used very frequently throughout the stack, and is in the center of the rear axle.
+The `base_link` frame is used very frequently throughout the stack, and is a projection of the rear-axle center onto the ground surface.
```
Comment (kenji-miyake) [inline docs/design/autoware-interfaces/components/vehicle-dimensions.md:7]: Although it's not directly related to this PR, I think the word `stack` can be changed to be more descriptive. Adding `Autoware` would be nice?

@xmfcx @LalithVipulananthan What do you think?

```suggestion
The `base_link` frame is used very frequently throughout the Autoware stack, and is a projection of the rear-axle center onto the ground surface.
```

**Review by LalithVipulananthan — COMMENTED** [review]
Looks fine to me, but one of the other reviewers should check the wording and approve.

@YamatoAndo, @mitsudome-r, @xmfcx - could you review the proposed change and confirm whether it is acceptable or not please?

**Review by YamatoAndo — APPROVED** [review]
Sorry for the late reply.
LGTM.

**Comment by xmfcx** [comment]
@isamu-takagi @mitsudome-r @YamatoAndo I've just realized that base_link was actually projected to the ground. Is there a reason why this convention is used instead of center of the rear axle?

**Comment by isamu-takagi** [comment]
@xmfcx 
I think the definition used when Autoware was first released are used as it is. 
I don't know how it came about, but maybe from following benefits:
- Since the height is often calculated based on the ground, there is no need to calculate the wheel radius.

**Pull request merged**